app/jobs/inboxes/fetch_imap_emails_job.rb

```diff
@@ -85,7 +90,7 @@ def fetch_message_ids_with_sequence(imap_client, channel)
       end
 
       batch_message_ids.each do |data|
-        message_id = build_mail_from_string(data.attr['BODY[HEADER.FIELDS (MESSAGE-ID)]']).message_id
+        message_id = build_mail_from_string(data.attr['BODY.PEEK[HEADER]']).message_id
```

Comment:
```suggestion
        message_id = build_mail_from_string(data.attr['BODY.[HEADER]']).message_id
```
While testing, emails weren't getting created on my local machine.  I believe this is different from the current deployment.